configs/spvcnn/README.md

```diff
@@ -0,0 +1,44 @@
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/72679458/226509154-80c27d8e-c138-426a-b92e-72846997b5b3.png" width="800"/>
+</div>
+
+## Introduction
+
+We implement SPVCNN with TorchSparse backend and provide the result and checkpoints on SemanticKITTI datasets.
+
+## Results and models
+
+### SemanticKITTI
+
+|   Method   | Lr schd | Mem (GB) | mIoU |                                                                                                                                  Download                                                                                                                                   |
+| :--------: | :-----: | :------: | :--: | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+| SPVCNN-W16 |   15e   |   3.9    | 61.9 | [model](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/spvcnn/spvcnn_w16_8xb2-15e_semantickitti_20230321_011645-a2734d85.pth) \| [log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/spvcnn/spvcnn_w16_8xb2-15e_semantickitti_20230321_011645.log) |
+| SPVCNN-W20 |   15e   |   4.2    | 62.7 | [model](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/spvcnn/spvcnn_w20_8xb2-15e_semantickitti_20230321_011649-519e7eff.pth) \| [log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/spvcnn/spvcnn_w20_8xb2-15e_semantickitti_20230321_011649.log) |
+| SPVCNN-W32 |   15e   |   5.4    | 64.3 | [model](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/spvcnn/spvcnn_w32_8xb2-15e_semantickitti_20230308_113324-f7c0c5b4.pth) \| [log](https://download.openmmlab.com/mmdetection3d/v1.0.0_models/spvcnn/spvcnn_w32_8xb2-15e_semantickitti_20230308_113324.log) |
+
+**Note:** We follow the implementation in SPVNAS original [repo](https://github.com/mit-han-lab/spvnas) and W16\\W20\\W32 indicates different number of channels.
+
+**Note:** Due to TorchSparse backend, the model performance is relatively dependent on random seeds, and if random seeds are not specified the model performance will be different (± 1.5 mIoU).
```

Comment:
The model performance is unstable with TorchSparse backend and may fluctuate by about 1.5 mIoU for different random seeds.